backend/controller/dal/async_calls.go

```diff
@@ -115,7 +115,7 @@ func (d *DAL) AcquireAsyncCall(ctx context.Context) (call *AsyncCall, err error)
 		return nil, fmt.Errorf("failed to parse origin key %q: %w", row.Origin, err)
 	}
 
-	decryptedRequest, err := d.decrypt(encryption.AsyncSubKey, row.Request)
+	decryptedRequest, err := d.decrypt(encryption.AsyncSubkey{}, row.Request)
```

Comment:
Unfortunately this is going to conflict with https://github.com/TBD54566975/ftl/pull/2400 once that's merged.

Reply:
I will revert those lines!